UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/ItemMatrixSystemInit.cs

```diff
@@ -9,12 +11,15 @@
 namespace TileMap.Behaviours
 {
 
-	public abstract class ItemMatrixSystemInit : NetworkBehaviour, IInitialiseSystem
+	public abstract class ItemMatrixSystemInit : NaughtyNetworkBehaviour, IInitialiseSystem
```

Comment:
oh? you know you could just modify the Mirror side, Since I don't know why There is a custom inspector for NetworkBehaviour

Reply:
not going to re-download mirror just to do that

Reply:
nah, is not that much andhhave a custom version of NetworkBehaviour seems messy